pkg/configv1/config.go

```diff
@@ -104,30 +105,10 @@ func (c *Config) init(kind Kind, apiVersion string) error {
 	c.APIVersion = apiVersion
 
 	switch kind {
-	case KindApplication:
+	case KindApplication, KindKubernetesApp, KindTerraformApp, KindCloudRunApp, KindLambdaApp, KindECSApp:
```

Comment:
To have backward compatibility, we treat them as the same as the KindApplication.